src/main/resources/db/migration/V2.1__tickets_url_nulable.sql

```diff
@@ -0,0 +1,6 @@
+-- alter EVENTS
+
+ALTER TABLE EVENTS ADD COLUMN NEW_TICKETS_URL VARCHAR(255);
+UPDATE EVENTS SET NEW_TICKETS_URL=TICKETS_URL;
+ALTER TABLE EVENTS DROP COLUMN TICKETS_URL;
+RENAME COLUMN EVENTS.NEW_TICKETS_URL TO TICKETS_URL;
```

Comment:
Why not just do: 
```
ALTER TABLE EVENTS MODIFY TICKETS_URL VARCHAR(255);
```

https://stackoverflow.com/questions/212939/how-do-i-modify-a-mysql-column-to-allow-null

Reply:
Syntax error: Encountered "MODIFY" at line 1, column 20.

`MODIFY` is not supported